tests/conftest.py

```diff
@@ -14,6 +15,31 @@
     LEAN4_EXAMPLE_URL,
 ]
 
+TEST_GITHUB_COMMIT_HASH = "3f8c5eb303a225cdef609498b8d87262e5ef344b"
```

Comment:
This line is not needed?

Reply:
yes, it should be passed to the tests as follows:

```py
def test_local_with_commit(clean_clone_and_checkout, lean4_example_url, local_test_path, example_commit_hash):
    # Initialize GitHub repo
    github_repo = LeanGitRepo(url=lean4_example_url, commit=example_commit_hash)
```

I'll revise and add new tests later.